mat64/fao_data_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+// Based on the SingularValueDecomposition class from Jama 1.0.3.
```

Comment:
Ha! That is still on a variety of the files. I'll at least remove this one.

#445